src/uwtools/drivers/driver.py

```diff
@@ -43,9 +55,9 @@ def requirements(self) -> None:
         """
 
     @abstractmethod
-    def resources(self) -> None:
+    def resources(self, platform: dict) -> Mapping:
         """
-        ???
+        Set up the formatting for the scheduler batch script.
```

Comment:
Perhaps leave this one as-is with `???` so that we can recognize it as something we need to handle explicitly later when we move beyond the MVP?